sdks/RepoToolset/tools/Build.proj

```diff
@@ -29,15 +30,23 @@
   <!--
     Default values. 
   -->
+  <ItemGroup>
+    <_ProjectToBuild Include="$(Projects)"/>
```

Comment:
Where is the correct place for a repo using this toolset to define these Projects?

Reply:
It's passed in as parameter from `build.ps1`. `build.ps1` takes an optional `-solution` parameter that can specify it. If the parameter is not passed we use `RepoRoot\*.sln`.

Reply:
It would be probably good to have some kind of mechanism how to specify these in a file somewhere in the repo. I'm open to proposals how to do this. Most repos on RepoToolset have a single solution in the repo root.

Reply:
I think we could set a property in the root Directory.Build.props file that identifies the root project(s).

Reply:
That's too late. Directory.Build.props are imported way too late. It's much easier to check for an optional file of well-known name in the repo root. For example, `Solutions.props` or something like that. 
Have you looked into https://github.com/Microsoft/MSBuildSdks/tree/master/src/Traversal? Could that work?

Reply:
>That's too late. Directory.Build.props are imported way too late. It's much easier to check for an optional file of well-known name in the repo root. For example, Projects.props or something like that.

OK I will take a deeper look at the ordering to see if there is a good place for it. 

> Have you looked into https://github.com/Microsoft/MSBuildSdks/tree/master/src/Traversal? Could that work?

Not in any detail yet but it is on my list.